src/plugins/controls/public/options_list/components/options_list_control.tsx

```diff
@@ -83,22 +79,38 @@ export const OptionsListControl = ({ typeaheadSubject }: { typeaheadSubject: Sub
       selectionDisplayNode: (
         <>
           {exclude && (
-            <EuiTextColor color="danger">
-              <b>{OptionsListStrings.control.getNegate()}</b>{' '}
-            </EuiTextColor>
+            <>
+              <span className="optionsList__negateLabel">
+                {existsSelected
+                  ? OptionsListStrings.control.getExcludeExists()
+                  : OptionsListStrings.control.getNegate()}
+              </span>{' '}
```

Comment:
Tried to add this space using CSS `:after`, but then this messes with the screen reader because the title of the button becomes `"DOES NOTExist"`- so, need to add the space explicitly here instead.